src/app/views/route-search-results/route-search-results.component.html

```diff
@@ -1,41 +1,126 @@
 <app-nav-bar title="Results"></app-nav-bar>
 
 <div class="big-navigation-gap">
-  <div class="route-search-results__route-information">
-    <div>
+  <div class="route-search-results__route-information center-center">
+    <div class="center-center">
       <div>A</div>
       <div>B</div>
     </div>
-    <div>
-      <div>{{ routeS.getMainFrom(route) }}</div>
-      <div>{{ routeS.getMainTo(route) }}</div>
+    <div class="route-search-results__nowrap-hidden center-center">
+      <div>
+        <span>{{ routeS.getMainFrom(route) }}</span>
+        <span class="route-search-results__location-secondary">, {{ routeS.getSecondaryFrom(route) }}</span>
+      </div>
+      <div>
+        <span>{{ routeS.getMainTo(route) }}</span>
+        <span class="route-search-results__location-secondary">, {{ routeS.getSecondaryTo(route) }}</span>
+      </div>
+    </div>
+    <div class="center-center">
+      <span (click)="handleBackToSearchClick()">
+        <fa-icon [icon]="iconS.getIcon('edit')"></fa-icon>
+      </span>
     </div>
-    <div><span (click)="handleBackToSearchClick()">change</span></div>
   </div>
 
   <ngb-accordion [closeOthers]="true" #acc="ngbAccordion">
 
     <ngb-panel *ngFor="let route of getSortedRoute()" id="{{route.vehicleId}}">
 
       <ng-template ngbPanelHeader>
-        <div class="route-search-results__item-header"
+        <div class="route-search-results__item-header center-center"
              (click)="acc.toggle(route.vehicleId)">
           <div>
+            <div class="route-search-results__column-name"> </div>
             <app-vehicle-icon [vehicle]="routeS.getVehicleIconString(route)"></app-vehicle-icon>
-            <div>{{ getCustomVehicleName(route) }}</div>
+            <div class="route-search-results__column-name">{{ getCustomVehicleName(route) }}</div>
+          </div>
+          <div *ngIf="routeS.getCo2Kilograms(route); else elseBlock">
+            <div class="route-search-results__column-name">
+              <span>
+                <fa-icon *ngFor="let i of getClouds(route)" [icon]="iconS.getIcon('cloud')"></fa-icon>
+              </span>
+            </div>
+            <div>
+              {{ routeS.getCo2Kilograms(route) | number: '1.2-2' }} kg
+            </div>
+          </div>
+          <ng-template #elseBlock>
+            <div class="route-search-results__green">
+              <fa-icon [icon]="iconS.getIcon('leaf')"></fa-icon>
+            </div>
+          </ng-template>
+          <div>
+            <div>
+              <span class="route-search-results__column-name">
+                <fa-icon [icon]="iconS.getIcon('clock')"></fa-icon>
```

Comment:
the clock icon doesnt really alight with the time value

<img width="106" alt="Screen Shot 2020-10-29 at 18 12 02" src="https://user-images.githubusercontent.com/16033930/97608203-52740200-1a12-11eb-9aab-ae31b2dccda7.png">